dist/manifest.json

```diff
@@ -33,7 +33,10 @@
       "https://superuser.com/*",
       "https://askubuntu.com/*",
       "http://stackoverflow.com/*",
-      "https://stackoverflow.com/*"
+      "https://stackoverflow.com/*",
+      "http://package.elm-lang.org/*",
```

Comment:
I think this site does load some of the content through javascript. Though, it seemed to be working for me:
<img width="718" alt="screen shot 2017-05-01 at 9 02 22 pm" src="https://cloud.githubusercontent.com/assets/2635560/25603595/ec199cee-2eb1-11e7-9a04-6175d08647f8.png">
I'll remove this for now. To handle sites like this, maybe we could implement some kind of `mouseover` delegate for `pre` to handle dynamic content? Instead of immediately appending the button to all `pre` elements on load.